src/Umbraco.Web.BackOffice/Controllers/MemberController.cs

```diff
@@ -149,134 +187,118 @@ public MemberListDisplay GetListNodeDisplay(string listName)
         /// <summary>
         /// Gets the content json for the member
         /// </summary>
-        /// <param name="key"></param>
-        /// <returns></returns>
+        /// <param name="key">The Guid key of the member</param>
+        /// <returns>The member for display</returns>
         [OutgoingEditorModelEvent]
         public MemberDisplay GetByKey(Guid key)
         {
-            var foundMember = _memberService.GetByKey(key);
+            // TODO: this is not finding the key currently after member creation
```

Comment:
Issue that I need to debug, straight after member creation this was throwing an exception. Looking into it.

Reply:
No longer happening